docs/books/admin_guide/05-vi.md

```diff
@@ -381,11 +381,11 @@ To switch to *Ex* mode, from *command* mode, type <kbd>:</kbd>.
 
 `?string`
 
-* Go to the next occurrence found:
+* Find the next matching string:
 
 <kbd>n</kbd>
 
-* Go to the previous occurrence found:
+* Find the last matching string:
```

Comment:
@jimcat8 I think "previous" is a better word than "last" in this case. "last" intimates the very last occurrence of the string, rather than the one that occurred before. I'd be happy with "Find the previous matching string:"